.github/workflows/ci.yml

```diff
@@ -149,9 +149,8 @@ jobs:
             sudo apt-get install -y capnproto libcapnp-dev
 
       - uses: actions/checkout@v4
-      - uses: dtolnay/rust-toolchain@master
+      - uses: dtolnay/rust-toolchain@nightly
         with:
-          toolchain: nightly-2025-05-13
```

Comment:
In https://github.com/capnproto/capnproto-rust/pull/458#discussion_r1412855424 we decided that it was preferable for the nightly to be pinned, so that CI does not spontaneously break.